### PR TITLE
Remove legjugglite

### PR DESCRIPTION
**Pull request by Beherith**
Remove as I disagree with such dilution. Please write any opinion you have on the matter as comment here.

### CODE REVIEW

**Comment by TheSilverHornet** [comment]
I can't see that there is anything worse about this unit concept than many or even most scav/'extra' units.  It makes no sense to single it out, especially as scav unit guidance was always inclusion positive.

**Comment by Beherith** [comment]
Taking one of the most iconic models of the base roster, chopping its head off and putting it on a chassis is far above my threshold. 

Considering that this unit is also included on both the cortex and legion roster, it does not get the leeway that scavenger (which aren't player facing) units receive. In fact the entire modOptions.experimentalextraunits needs a rethink, of whether they should be easily player-accessible. My thought would be these units should be reserved as a 'reward' for good plays where a player manages to get it via resurrecting the scavenger unit, or via acquiring a scavenger constructor. 

"Extra Units" can be interpreted as "Units that we intend to include by default, or just didn't make the cut". You can take away toys after you've given them to players.

**Comment by StanczakDominik** [comment]
Removing this would decrease the number of puns in the game, which I'm opposed to on ideological grounds :P speaking of, you forgot to remove the language json entries.

I get that you're protective of the Jugg model, but at the same time I do think it makes perfect sense as a Scav unit. Isn't the root of the problem that, if you just make a Raptors lobby and enable every single boolean option in advanced settings, as is tradition in these lobbies (and I do still intend to look through that data export!), you really can't tell which units comes from where? We get a ton of questions in #academy-chat like "hey, when do you usually switch to t2 nanos?" or "hey, why can't my dragons breathe fire anymore?", and the answer is usually "you've been playing in a lobby with crazy modoptions".

> Considering that this unit is also included on both the cortex and legion roster, it does not get the leeway that scavenger (which aren't player facing) units receive. In fact the entire modOptions.experimentalextraunits needs a rethink, of whether they should be easily player-accessible. My thought would be these units should be reserved as a 'reward' for good plays where a player manages to get it via resurrecting the scavenger unit, or via acquiring a scavenger constructor.

I really agree with this take, but if we get that resolved, I think the jugglite could find a place there. In fact if that's the case, the only reaction from new players that I'd imagine is "oooh, it's the Jugg's head! They cut the Jugg's head off!" because it's so recognizable. But that's not a bad thing, is it? Helps get the idea of what Scavs are across, which to me is a nice bit of worldbuilding.

So my ideal-world scenario resolution for this that hopefully keeps everyone happy would be:
1. Revert the file removal
2. Temporarily drop the jugglite entries from the modoption-gated Leg and Cor gantries.
    - This should still let Scavs spawn them, right? Do they even spawn them atm? Should they?
4. Figure out how we want the extra units modoptions to work
5. Have these be a reward for good plays as mentioned above

**Comment by icexuick** [comment]
I fully agree with Beherith.
There should be more than enough room to experiment / try-out new fun units, that don't involve just putting a head on a chassis. This takes away so much from the original unit and adds little to none in return.

I think we need to keep respecting the unique units much more. Seeing the use of the Juggernaut laser-weapon + sound on multiple other units is just taking that awesome-unique-factor away from the Juggernaut. I much more appreciate the new weapons/ideas on f.e. the Starfall, Phoenix/Bastion are great examples.

**Comment by drivver44** [comment]
personally i think it would be best to close the pr with no merge and have an issue raised instead 
i dont see destructive progress working here as it creates more problems associated with semi relevant topics of github permissions

**Comment by TheSilverHornet** [comment]
When the 'extra units' modption is enabled, all extra units that are cortex are also added to legion currently, that's nothing new, nor unique to this unit. Eg epic karg is also in legion gantry, as is epic grunt, etc.

The 'extra unit' pack having a rethink is not something I am opposed to, quite the opposite, as I've said elsewhere it could do with a divide of 'sane' and 'insane' along some lines, and is something I want to do at some point.  There's a fairly clear divide of units that a lot of players would want in a game, and some that are stopping them enabling the pack as a whole, the most obvious example being pawn launcher, but a lot of the 'epics' are in the same bucket, eg ratte, or epic pulsar.

I'd be more than fine with a discussion on that, probably an issue, or discord thread, but that is not the same thing at all as a PR to essentially "remove this unit in particular now, just because it's new".

As last I understood Damgams goals with scavs, it will be impossible to resurrect their units at all, cons or otherwise, so the 'extra unit' modoption will be the only way they can be played with by players.

This unit being a recognisable mashup is sort of the point however, that's exactly what a scav unit is meant to be. They essentially are autonomous bits of the scrapheap, the player seeing that the scavs have salvaged a juggernaut is completely in keeping with scav lore.  When this unit concept was discussed with Damgam ages ago, up to when merged by him, there was at no point suggestion that the unit was anything other than a very good fit for scavs.

Icewuick, the units you mentioned are main game units such as a Brute or Starlight would be, albeit for Legion. They're not scav units, so comparison is fairly meaningless - in fact given scav unit design is essentially 'scrapheap kitbash' as said above, it could even be considered less in keeping if it were an entirely unique model.

**Comment by Beherith** [comment]
Yes the unit has a role, but that is independent of the model and irrelevant to the pr. 

The complete PR stands along with file removal to avoid it getting tweakdef'd back in. 

It does fit the scav lore however.

How are GH permissions relevant here, driver?

**Comment by drivver44** [comment]
> How are GH permissions relevant here, driver?

in short its that the people who got these units in are not the people who's job it is to ensure that the quality we want is being enforced
if this is something that should of been rejected before being added to the game 
we dont have review processes outside of someone just merging stuff at times
so if these units are an issue we also need to have a process to ensure that these units do not get merged into bar 
which means we should be changing how gh permissions  are handled

**Comment by TheSilverHornet** [comment]
To the latter, the objection seems to be 'scav design' rather than units. This unit is in keeping with scav design.

It could very well be worth rethinking that in some way, especially as scavs will inherit old Legion units, but that seems like a problem to worry about then, not now, and certainly not piecemeal.

**Comment by Beherith** [comment]
> > How are GH permissions relevant here, driver?
> 
> in short its that the people who got these units in are not the people who's job it is to ensure that the quality we want is being enforced if this is something that should of been rejected before being added to the game we dont have review processes outside of someone just merging stuff at times so if these units are an issue we also need to have a process to ensure that these units do not get merged into bar which means we should be changing how gh permissions are handled
> 
> tldr if the units are an issue so is the process that got us here is an issue aswell

There is nothing wrong with retrospection in any case :)

**Comment by TheSilverHornet** [comment]
Beherith, could you explain why you feel this unit in particular is more wrong than other scav units that are also mashups?

**Comment by Beherith** [comment]
Because it takes the sights and sounds of one of the most iconic base units and trivializes it in a way that I feel dilutes the original model's character.

**Comment by TheSilverHornet** [comment]
Thankyou for clarifying. However, scavs by definition are scavengers, raiding dead units for parts and recycling. It being recognisably from a juggernaut is surely in keeping with scav lore? As Perfi said before, and as I intended, it's actually a tangible lore moment in game that players can see what it is/was.

If I'd used a starlight head, or say a big bertha/basilica, would that be as objectionable?  Or if the jugg head was clearly in more scrapped state, eg lacked side panels, you could see loose wires and internal cogs, and such?'

**Comment by StanczakDominik** [comment]
For an iconic unit, game play wise it seems to be in a decently bad place - why would you make a juggernaut when a lolcannon or mass t2 bomber raid will end the game faster and without risking blowing up your entire army?

In that sense, I'm glad that at least the head gets to, ahem, see some action 😜

**Comment by TheSilverHornet** [comment]
If objection is solely the juggernaut origin, I can just rework the unit to use some other wreckage as a donor.  Given that it needs to be long range damage, starlight would make the most sense.

I could also lean more into the premise and use multiple weapons/components, but that would probably bleed it out of the role it has to fill, and make it too generalist.

**Comment by Beherith** [comment]
I would very much prefer original work in lieu of kitbashes in general.
